pom.xml

```diff
@@ -62,4 +62,12 @@
         </profile>
 
     </profiles>
+
+     <dependencies>
+        <dependency>
+            <groupId>org.technologybrewery.habushu</groupId>
+            <artifactId>habushu-maven-plugin</artifactId>
+            <version>2.10.1-SNAPSHOT</version>
```

Comment:
It might be a good idea to add a property for the version in pom file since I see this changing quite a bit!

Reply:
Under normal circumstances a variable would make sense but this particular instance is the only instance of habushu in orphedomos.